New version: IntervalOptimisation v0.5.0

JuliaRegistrator opened the pull request:
- Registering package: IntervalOptimisation
- Repository: https://github.com/JuliaIntervals/IntervalOptimisation.jl
- Created by: @OlivierHnt
- Version: v0.5.0
- Commit: ca926f6cfad29eab43e76fd08b5fd314ad3c6bbc
- Reviewed by: @OlivierHnt
- Reference: https://github.com/JuliaIntervals/IntervalOptimisation.jl/commit/ca926f6cfad29eab43e76fd08b5fd314ad3c6bbc#commitcomment-165357479
- Description: Rigorous global optimisation
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- Compatibility with IntervalArithmetic 0.22
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd61dbcb18e8a3938e96eb91b0828ad34d3e64c445a049e24958a8babdfbb3602f9f30bc9371895a07d86110607c3f228618a413a8adf6a2d85db8ee77893e9edbdcee7f94739a8ce7ff8919dbacbc51b69daaf86af09277188cce930ce97e883fc73bb608eb3734d9ec8fbff79c2f676a49c56c0cf11f1609bcd03506559dd56f59a50cc5595887bd14e568861fed8a5be60ab60f01167a0a40df3706804b321bc7dc9124b2e44381910fb9bed3ed615aebeedb7b08144c7ae543ee0ba4ada40a2 -->

OlivierHnt commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- Compatibility with IntervalArithmetic 0.22

OlivierHnt commented:
[noblock]

JuliaRegistrator committed:
New version: IntervalOptimisation v0.5.0

UUID: c7c68f13-a4a2-5b9a-b424-07d005f8d9d2
Repo: https://github.com/JuliaIntervals/IntervalOptimisation.jl.git
Tree: d84d0c02f7b202dcd157bd1e9a320156e4f3a23c

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
